makroud_test.go

```diff
@@ -3410,6 +3401,32 @@ func Setup(t require.TestingT, options ...makroud.Option) SetupCallback {
 	}
 }
 
+func Options(options ...makroud.Option) []makroud.Option {
+	dbOpts := []makroud.Option{
+		dbParamString(makroud.Host, "host", "PGHOST"),
+		dbParamInt(makroud.Port, "port", "PGPORT"),
+		dbParamString(makroud.User, "user", "PGUSER"),
+		dbParamString(makroud.Password, "password", "PGPASSWORD"),
+		dbParamString(makroud.Database, "name", "PGDATABASE"),
+		makroud.Cache(true),
+	}
+	return append(dbOpts, options...)
+}
+
+func ClientOptions(options ...makroud.Option) *makroud.ClientOptions {
+	dbOpts := Options(options...)
+
+	cliOpts := makroud.NewClientOptions()
```

Comment:
`cliOpts` is ambiguous.

```suggestion
	clientOpts := makroud.NewClientOptions()
```

Or just:

```suggestion
	opts := makroud.NewClientOptions()
```